src/evm/protocol/vm/state.rs

```diff
@@ -507,15 +507,20 @@ mod tests {
         collections::{HashMap, HashSet},
         path::PathBuf,
         str::FromStr,
+        sync::Arc,
     };
 
-    use ethers::{prelude::H256, types::Address as EthAddress};
+    use ethers::{
+        prelude::{Http, Provider, H256},
+        types::Address as EthAddress,
```

Comment:
👀 👀 👀 👀 👀 👀 👀  eettheeeerrsssss?? I banneeedd yooouuu! you shall not return to this repo!!

Reply:
kidding, I will remove it again after this is merged 😂